app/src/test/java/com/github/se/travelpouch/ui/travel/TravelLogicUnitTest.kt

```diff
@@ -2,17 +2,16 @@ package com.github.se.travelpouch.ui.travel
 
 import com.google.firebase.Timestamp
 import java.text.ParseException
-import org.junit.Assert.*
 import org.junit.Test
 
 class ParseDateToTimestampTest {
 
   @Test
   fun testParseDateToTimestamp_validDate() {
     val dateString = "01/01/2023"
-    val expectedTimestamp = Timestamp(1_672_527_600, 0) // Expected timestamp for 01/01/2023 in CEST
+    Timestamp(1_672_527_600, 0) // Expected timestamp for 01/01/2023 in CEST
```

Comment:
I think there is a problem here because TimeStamp is not used

Reply:
I suppress the `val expectedTimestamp` because it was not used. We call Timestamp to verify that is not fail when we give a precise value.

Reply:
I guess it is valid. This helps us to see if we call TimeStamp correctly.